eventauth.go

```diff
@@ -313,6 +321,12 @@ func (a *AuthEvents) AddEvent(event *Event) error {
 	if event.StateKey() == nil {
 		return fmt.Errorf("AddEvent: event %q does not have a state key", event.Type())
 	}
+	for _, ev := range a.events {
+		if ev.RoomID() != "" && ev.RoomID() != event.RoomID() {
```

Comment:
Is `ev.RoomID() != ""` ever going to be true?

Reply:
Ah, a relic from other tests before. :D